### DISCUSSION TITLE
Custom Models are Gone After Upgrade

### DISCUSSION THREAD

**Discussion by HiProAI** [upscayl/upscayl#647] [Q&A] [answered]
I've upgraded a few times now and every time I upgrade it blows away the Resources>Models folder and installs a new one wit hthe basic files. I have donwnloaded and added the custom models so that I have extra upscaling models. Id suggest not blogin away the folder on upgrade install.  I'm using Windows 11 
I found out what was going on and recopied all the custom model upscale files after update this time and its all back.

**Comment by aaronliu0130** [comment, marked as answer]
See #648. (Please don't file bug reports in discussions, btw)
